src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -516,6 +524,122 @@ internal static SafeSslHandle AllocateSslHandle(SslAuthenticationOptions sslAuth
             return sslHandle;
         }
 
+        internal static string[] GetDefaultSignatureAlgorithms()
+        {
+            ushort[] rawAlgs = Interop.Ssl.GetDefaultSignatureAlgorithms();
+
+            static string ConvertAlg(ushort rawAlg) => rawAlg switch
+            {
+                0x0201 => "rsa_pkcs1_sha1",
```

Comment:
this probably needs a comment how did you get those values so it's easier to update if list changes (i.e. which OpenSSL header file, doc ref, IANA link)